.github/workflows/build-package.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Package
+
+env:
+  MAIN_BRANCH: "all-azure_gdpr"
+  PACKAGE_CLOUD_REPO_NAME: "citusdata/azure"
+  PACKAGE_CLOUD_API_TOKEN: ${{ secrets.PACKAGE_CLOUD_API_TOKEN }}
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+  PACKAGING_SECRET_KEY: ${{ secrets.PACKAGING_SECRET_KEY }}
+  PACKAGING_PASSPHRASE: ${{ secrets.PACKAGING_PASSPHRASE }}
+on:
+  push:
+    branches:
+      - "**"
+
+  workflow_dispatch:
+
+jobs:
+  build_package:
+    name: Build package
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - el/8
```

Comment:
Do we actually use el/8 or el/7 ?

Reply:
el/7 also exist in travis file. So I added el/7 as well